primitives/runtime-interface/src/impls.rs

```diff
@@ -54,6 +54,10 @@ pub fn pointer_and_len_to_u64(ptr: u32, len: u32) -> u64 {
 }
 
 /// Splits an `u64` into the pointer and length.
+///
+/// Runtime API functions return a 64-bit value which encodes a pointer in the least-significant
+/// 32-bits and a length in the most-significant 32 bits. This interprets the returned value as a pointer,
+/// length tuple.
 pub fn pointer_and_len_from_u64(val: u64) -> (u32, u32) {
```

Comment:
I think we should move this function into the lib.rs file. As this file should only contain implementations of the traits for types.

Reply:
Good point! Should've looked around more closely...